regularizer_new.py

```diff
@@ -60,35 +206,31 @@ def tokenizer_helper(s, add_special_tokens=True):
             else:
                 return self.tokenizer(s)
```

Comment:
I know this is technically not part of this PR, but please put nested functions such as `tokenizer_helper` at the top of the function if used.

Reply:
Changed.